keps/sig-api-machinery/1281-network-proxy/README.md

```diff
@@ -287,14 +288,14 @@ and will be routed based on the ‘cluster’ NetworkContext.
 - **Admission Webhooks**
 Admission webhooks can be destined for a service or a URL.
 If destined for a service then the service rules apply (send to 'cluster').
-If destined for a URL then we will use the ‘master’ NetworkContext.
+If destined for a URL then we will use the ‘controlplane’ NetworkContext.
 - **Aggregated API Server (and OpenAPI requests for aggregated resources)**
 Aggregated API Servers can be destined for a service or a URL.
```

Comment:
Is it true that Aggregated API Servers can be destined for a URL? I only see service in the [spec](https://kubernetes.io/docs/reference/kubernetes-api/cluster-resources/api-service-v1/#APIServiceSpec).

Reply:
I find no kube-apiserver support for Aggregated API destined for URL, and trimmed this.